src/magicbox/InputManager.ts

```diff
@@ -218,14 +218,15 @@ export class InputManager {
   private keydown(e: KeyboardEvent) {
     switch (e.keyCode || e.which) {
       case 9: // Tab key
-        if (!this.justPressedTab && this.magicBox.hasSuggestions()) {
-          e.preventDefault();
-        }
-        this.justPressedTab = true;
+        // Take care of not "preventing" the default event behaviour : For accessibility reasons, it is much simpler
```

Comment:
I wonder if this comment is needed. It is good context for this PR, but will it be useful to someone reading the code in 2 years from now?

Reply:
Well my reasoning was that the next "switch case" stop propagation and stop the default flow of the keyboard event, which is why I'm adding an explicit comment here that explains why it's not a good idea to do this for tabpress.